src/Stryker.Core/Stryker.Core/Reporters/JsonReporter/JsonMutant.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Stryker.Core.Reporters.Json
+{
+    public class JsonMutant
+    {
+        public int Id { get; set; }
+        public string MutatorName { get; set; }
+        public string Replacement { get; set; }
+        public JsonMutantLocation Location { get; set; }
+        public string Status { get; set; }
+
+        public class JsonMutantLocation
```

Comment:
Why is this a nested class? I think it should be a standalone class.

Reply:
I felt like this class has no purpose outside of the JsonMutant so should be as close to it as possible. It could be moved to it's own class.

Reply:
My preference is a JsomMutantLocation file, but this is okay too.